pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -632,12 +633,16 @@ func (p *PriorityQueue) MoveAllToActiveQueue() {
 		} else {
 			if err := p.activeQ.Add(pInfo); err != nil {
 				klog.Errorf("Error adding pod %v to the scheduling queue: %v", pod.Name, err)
+			} else {
+				moved = true
 			}
 		}
 	}
 	p.unschedulableQ.clear()
 	p.moveRequestCycle = p.schedulingCycle
-	p.cond.Broadcast()
+	if moved {
+		p.cond.Broadcast()
```

Comment:
I think this change seems reasonable, as @mikedanese mentioned, do we any see benefit of this change?

/cc @bsalamat